### ISSUE TITLE
Conflict with jsbeautify

### ISSUE THREAD

**Issue by vjeux** [prettier/prettier-vscode#454]
People keep complaining about this on twitter at a rate of one every few days, so it’s somewhat serious:

https://twitter.com/herbhagely/status/995468067172995072?s=21

If you have both jsbeautify and prettier then the jsx format gets screwed up.

It would be nice to warn about it somehow so that people don’t get confused and blame it on prettier :)

**Comment by lipis** [comment]
I think these are all related.. #453 #452 and I guess the #444 is trying to address that..

Giving a warning within the extension would be nice, but it could be annoying if the other extension is actually installed but ignoring let's say the `.js` files.. quite a tricky one.. Obviously we don't want to suggest for uninstalling the other extensions completely..

**Comment by capaj** [comment]
> Obviously we don't want to suggest for uninstalling the other extensions completely..

@lipis  Isn't prettier  meant to be the definitive solution for formatting code? I certainly think it is and I don't want to have any other code formatting plugins alongside it because all of them are light years behind prettier. I know it sounds harsh, but tell me this-do you have any other code formatting extensions installed on your machine?

**Comment by lipis** [comment]
I'm with you and yes I don't have any other formatters.. but letting the users know that there might be conflict is one thing.. and telling them to uninstall somebody else's hard work just doesn't seem right.

**Comment by mckernanin** [comment]
If you don't want to disable beautify, add this to your workspace or project config to make it ignore .js and .jsx files:

```
    "beautify.ignore": [
        "**/*.js",
        "**/*.jsx"
    ]
```

**Comment by lipis** [comment]
I think this one is solved in #444

**Comment by fullakingi** [comment]
this is a better fix: https://stackoverflow.com/questions/44993808/visual-studio-code-changes-format-react-jsx
